### DISCUSSION TITLE
can we have both vscode for m1 and intel in the same device.

### DISCUSSION THREAD

**Discussion by nibin-matteo-git** [Jyers/Marlin#939] [Q&A] [answered]
Just downloaded the vscode m1. But I already have the intel version installed. For some reason only the intel version shows up in the launcher. But I can find the m1 version in the applications folder in finder. 
I'm not sure if its because of this but every time I open the m1 version, it asks me to sign in to sync. I do it but the next time I open it, the same thing pops up again and I have to link it to my github account again.

**Comment by Jyers** [comment, marked as answer]
Going to mark this as answered so it's not sitting in the feed. Sorry you never got any confirmation on this, but I don't think I can be of much help...
